packages/studio-web/package.json

```diff
@@ -14,7 +14,7 @@
         "test": "ng test",
         "test:once": "ng test --watch=false --browsers ChromeHeadlessCI",
         "extract-i18n": "ng extract-i18n --format json --output-path src/i18n; echo '' >> src/i18n/messages.json",
-        "check-fr-l10n": "bash -c \"diff -w <(cat src/i18n/messages.json | sed 's/:.*//') <(cat src/i18n/messages.fr.json | sed 's/:.*//')\""
+        "check-fr-l10n": "bash -c \"diff -w <(cat src/i18n/messages.json | sed 's/:.*//' | sort) <(cat src/i18n/messages.fr.json | sed 's/:.*//' | sort)\""
```

Comment:
This is done in https://github.com/ReadAlongs/Web-Component/pull/118

Reply:
ah, thanks!  I didn't notice it there, I was just annoyed at having to manually move things around